editor/src/core/shared/template-path.ts

```diff
@@ -149,71 +158,59 @@ export function toString(target: TemplatePath): string {
   }
 }
 
-function newScenePath(elements: StaticElementPath): ScenePath {
+function newScenePath(elementPaths: StaticElementPath[]): ScenePath {
   return {
     type: 'scenepath',
-    sceneElementPath: elements,
+    sceneElementPaths: elementPaths,
   }
 }
 
-const emptyScenePath: ScenePath = newScenePath(([] as any) as StaticElementPath)
+export const emptyScenePath: ScenePath = newScenePath(([] as any) as StaticElementPath[])
```

Comment:
is the `as any) as StaticElementPath[]` still needed? maybe typescript became smarter since then

Reply:
Ha, it actually did become smarter! Nice :+1:

Reply:
Fixed